edxval/tests/test_api.py

```diff
@@ -1770,7 +1770,7 @@ def test_multiple_external_transcripts_for_language(self):
 
         # Verify transcript record is created with correct data i.e sub field transcript.
         expected_transcripts = [
-            dict(constants.VIDEO_TRANSCRIPT_CUSTOM_SRT, video_id=edx_video_id, language_code='en')
+            dict(constants.VIDEO_TRANSCRIPT_CUSTOM_SJSON, video_id=edx_video_id, language_code='en')
```

Comment:
Why is this changed?

Reply:
Because the behavior changed. Here's a more detailed explanation: 

First, we need to identify that:
- `sub_transcript_file_name` is an _srt_ video transcript
- `ext_transcript_file_name` is an _sjson_ video transcript

```
# Import xml with empty edx video id.
edx_video_id = api.import_from_xml(
    etree.fromstring('<video_asset/>'),
    '',
    self.file_system,
    constants.EXPORT_IMPORT_STATIC_DIR,
    {
        'en': [sub_transcript_file_name, ext_transcript_file_name]
    }
)
```

If we take a look at the import segment of code, shown above, we can see that for the same *language_code* was given two different files/transcripts.

**Previously**, if a transcript was already uploaded/imported for a certain video, then importing that same video with a new transcript did not overwrite it. So, the initially imported `sub_transcript_file_name` transcript does not get replaced, and thus the file remains an _srt_ transcript.

**Currently**, if a transcript was already uploaded/imported for a certain video, then importing the same video with a new transcript will overwrite it if the content is different. Since the `sub_transcript_file_name` and the `ext_transcript_file_name` transcripts have different file contents, then when the `ext_transcript_file_name` transcript was uploaded, the transcript got replaced and became an _sjson_ transcript. 

Accordingly, we had to update the test :+1:

I tried to keep it brief, so please let me know if you'd like me to go into more details :slightly_smiling_face:

Reply:
Thanks for the brief. I would say to update the test docstring to cover this change. It will be easier to follow in the future.